src/main/plugin/iso19139.rndt/loc/ita/strings.xml

```diff
@@ -210,6 +210,8 @@
   <onlineResource>Risorsa on-line</onlineResource>
   <name>Nome</name>
   <URL>URL</URL>
+  <dimensionName>Nome</dimensionName>
+  <dimensionSize>Misura della dimensione</dimensionSize>
```

Comment:
Serve anche una label "Risoluzione in metri"

Reply:
fixed